### PR TITLE
fix(i18n/es): fixed translation errors and translated missing keys

### PR DESCRIPTION
**Pull request by sawa-ko**
Closes # 

### What's changed

Some translation errors are corrected and new corrections are added.

### Notes to reviewers

I also noticed that some keys that I had already translated have been put back into English, it's weird, so I had to translate them again.

### CODE REVIEW

File: packages/hoppscotch-common/locales/es.json [hunk 1]
```diff
@@ -456,16 +456,16 @@
     "postman_environment": "Entorno de Postman",
     "postman_environment_description": "Importar entorno de Postman desde un archivo JSON",
     "title": "Importar",
-    "file_size_limit_exceeded_warning_multiple_files": "Chosen files exceed the recommended limit of 10MB. Only the first {files} selected will be imported",
-    "file_size_limit_exceeded_warning_single_file": "The currently chosen file exceeds the recommended limit of 10MB. Please select another file.",
-    "success": "Successfully imported"
+    "file_size_limit_exceeded_warning_multiple_files": "Los archivos seleccionados exceden el límite recomendado de 10MB. Sólo se importarán los primeros {ficheros} seleccionados.",
```
Comment (nivedin) [inline packages/hoppscotch-common/locales/es.json:459]:
```suggestion
    "file_size_limit_exceeded_warning_multiple_files": "Los archivos seleccionados exceden el límite recomendado de 10MB. Sólo se importarán los primeros {files} seleccionados.",
```

should not change the variable name

**Review by nivedin — CHANGES_REQUESTED** [review]

File: packages/hoppscotch-common/locales/es.json (hunk 1 above)
Comment (sawa-ko) [reply packages/hoppscotch-common/locales/es.json]: Oh yes, sorry

File: packages/hoppscotch-common/locales/es.json [hunk 2]
```diff
@@ -139,33 +139,33 @@
       "redirect_no_client_id": "No se ha definido el ID de cliente",
       "redirect_no_client_secret": "No se ha definido ningún ID secreto de cliente",
       "redirect_no_code_verifier": "No se ha definido ningún verificador de códigos",
-      "redirect_no_token_endpoint": "No se ha definido ningún punto final de token",
+      "redirect_no_token_punto final": "No se ha definido ningún punto final de token",
```
Comment (nivedin) [inline packages/hoppscotch-common/locales/es.json:142]: keep the key name same

File: packages/hoppscotch-common/locales/es.json [hunk 3]
```diff
@@ -139,33 +139,33 @@
...
-      "label_username": "Username",
-      "label_auth_code": "Authorization Code",
-      "label_client_credentials": "Client Credentials"
+      "token_generation_oidc_discovery_failed": "Fallo en la generación del token: Error en el descubrimiento de OpenID Connect",
+      "grant_type": "Tipo de autorización",
+      "grant_type_auth_code": "Código de autorización",
+      "token_fetched_successfully": "Token recuperado correctamente",
+      "token_fetch_failed": "Fallo al recuperar el token",
+      "validation_failed": "Fallo de validación, comprueba los campos del formulario",
+      "label_authorization_punto final": "Punto final de autorización",
```
Comment (nivedin) [inline packages/hoppscotch-common/locales/es.json:151]: keep the key name same

File: packages/hoppscotch-common/locales/es.json [hunk 4]
```diff
@@ -139,33 +139,33 @@
...
+      "label_client_id": "ID de cliente",
+      "label_client_secret": "Secreto de cliente",
+      "label_code_challenge": "Código de desafío",
+      "label_code_challenge_method": "Método código de desafío",
+      "label_code_verifier": "Verificador de código",
+      "label_scopes": "Ámbitos",
+      "label_token_punto final": "Punto final de token",
```
Comment (nivedin) [inline packages/hoppscotch-common/locales/es.json:158]: keep the key name same

File: packages/hoppscotch-common/locales/es.json [hunk 5]
```diff
@@ -252,12 +252,12 @@
...
   "empty": {
-    "authorization": "Esta solicitud no utiliza ninguna autorización.",
+    "authorization": "Esta solicitud no utiliza ninguna autorización",
     "body": "Esta solicitud no tiene cuerpo",
     "collection": "Colección vacía",
     "collections": "No hay colecciones",
-    "documentation": "Conectarse a un punto final de GraphQL para ver la documentación",
-    "endpoint": "El punto final no puede estar vacío",
+    "documentation": "Es necesario conectarse a un punto final de GraphQL para ver la documentación",
+    "punto final": "El punto final no puede estar vacío",
```
Comment (nivedin) [inline packages/hoppscotch-common/locales/es.json:260]: keep the key name same

**Review by nivedin — CHANGES_REQUESTED** [review]
there are multiple strings with key name changed, please revert them

**Comment by sawa-ko** [comment]
Oh no, my "replace everything" went wrong. I'll fix everything. 🥲

**Review by nivedin — APPROVED** [review]

**Review by jamesgeorge007 — APPROVED** [review]

**Pull request merged**